samples/ResilienceStrategy/Retry/Retry.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Polly" />
```

Comment:
Not sure about this. Shall we use the central package management even for Samples?

Reply:
As it's becoming more mainstream I would suggest yes as it's "best practice" and it also makes our lives easier to maintain the samples as it's one file to update, not X Files (👽).